### DISCUSSION TITLE
Emulation of other peripherals (not just keyboard, mouse, network, joystick or things like that)

### DISCUSSION THREAD

**Discussion by ACompExp** [86Box/86Box#1732] [Ideas]
I want to get emulation of some more peripherals, not just mouse, keyboard, joystick, network, etc. I am requesting a feature where we can emulate printing through parallel port. 
I also want emulation of webcam and microphone (again, through parallel/serial port).

If these peripherals were added, then 86Box would be considered as one of the best PC emulators ever (i mean, i would consider it).

**Comment by ts-korhonen** [comment]
Parallel port printer emulation is already implemented.

**Comment by maxdlpee** [comment]
> I want to get emulation of some more peripherals, not just mouse, keyboard, joystick, network, etc.

_Proceeds to not specify said peripherals_

> I am requesting a feature where we can emulate printing through parallel port.

Except it is already a thing?

> I also want emulation of webcam and microphone (again, through parallel/serial port).

See, 86Box can't emulate what does not exist. Besides, even if those sorts of peripherals existed, there just is no demand for those.

**Comment by ts-korhonen** [comment]
> 86Box can't emulate what does not exist

Parallel port webcams do exist, not so sure about microphones.

I would be good to be more specific about what devices exactly is meant.

**Reply by Torinde** [reply]
He lists:
- [x] printing
- [ ] #3872, but on parallel/serial port
- camera 
  - [ ] [on parallel port](https://www.edaboard.com/threads/parallel-port-webcam-interfacing-in-ms-dos-using-c-language.36092/)
  - [ ] on USB would also be useful, which probably would enable also "[other peripherals](https://github.com/86Box/86Box/discussions/2907#discussioncomment-4273100)"

**Comment by OBattler** [comment]
I'm converting this to a discussion.
